mobile/lib/app/features/login/view/login_form.dart

```diff
@@ -6,23 +6,25 @@ import 'package:triggo/app/routes/routes_names.dart';
 import 'package:triggo/app/widgets/button.triggo.dart';
 import 'package:triggo/app/widgets/input.triggo.dart';
 
-class PasswordInputScreen extends StatelessWidget {
-  const PasswordInputScreen({super.key});
+class LoginForm extends StatelessWidget {
+  const LoginForm({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(),
-      body: Padding(
-        padding: const EdgeInsets.all(16.0),
-        child: BlocListener<LoginBloc, LoginState>(
-          listener: _listener,
+    return BlocListener<LoginBloc, LoginState>(
+      listener: _listener,
+      child: Align(
+        child: SingleChildScrollView(
+          padding: const EdgeInsets.all(16),
           child: Column(
+            mainAxisSize: MainAxisSize.min,
             children: [
-              _PasswordLabel(),
-              const SizedBox(height: 20),
+              _Label(),
+              const SizedBox(height: 12),
+              _EmailInput(),
+              const SizedBox(height: 12),
               _PasswordInput(),
-              const SizedBox(height: 20),
+              const SizedBox(height: 12),
               _LoginButton(),
```

Comment:
Even the fact that I'll not request you for that, I think there's a widget that allows to put the same height between its items, I don't have the name atm, but I'll let you find it if you want

Reply:
Mmmhhh yep, I think you're right. I'll search into that.

Reply:
I find the `Warp` class where we can add spacing. But we our custom Input it doesn't really apply so I'll search into that later